### ISSUE TITLE
Testing on new data!

### ISSUE THREAD

**Issue by sicarioakki** [cardwing/Codes-for-Lane-Detection#149]
Hey! @cardwing 
How to test the pre-trained model on other test data (apart from CULane or any other datasets mentioned in this repo)?
Are the ground truth labels required??

**Comment by harryhan618** [comment]
@sicarioakki I think anwsers can be found in `ENet-Label-Torch/testLane.lua`. GT lables are not required.
